app/models/mpi_data.rb

```diff
@@ -236,6 +236,7 @@ def add_ids(response)
     profile.birls_id = response.parsed_codes[:birls_id].presence
     profile.participant_id = response.parsed_codes[:participant_id].presence
 
+    profile.participant_ids = profile.participant_ids.push(profile.participant_id).uniq
```

Comment:
Does this line actually do anything? When accessing `target_veteran.participant_ids` later, the PID from the ICN lookup isn't in that list (hence the `push` [here](https://github.com/department-of-veterans-affairs/vets-api/pull/12087/files#diff-fb7560ef34123fc19745681e79b7e6a637fd6dc9302506824ee54d067baff42dR64)).

Reply:
I confirmed that we achieve the goal without this addition

Reply:
what's the decision here? do we need this?